{
  "name": "@arabasta/eslint-plugin-report-caught-error",
  "version": "1.0.6",
  "type": "commonjs",
  "description": "ESLint plugin that enforces caught errors in try/catches to be reported",
  "author": "Stoyan Kolev",
  "repository": {
    "type": "git",
    "url": "https://github.com/CloudNStoyan/arabasta"
  },
  "types": "./src/index.d.ts",
  "main": "./src/eslint-plugin-report-caught-error.js",
  "exports": {
    "./package.json": "./package.json",
    ".": {
      "types": "./src/index.d.ts",
      "default": "./src/eslint-plugin-report-caught-error.js"
    }
  },
  "scripts": {
    "test": "vitest --globals --dir ./src/rules/"
  },
  "devDependencies": {
    "eslint": "^9.0.0",
    "vitest": "^1.4.0"
  },
  "peerDependencies": {
    "eslint": ">=8.0.0"
  },
  "license": "MIT",
  "keywords": [
    "eslint",
    "eslint-plugin",
    "eslintplugin",
    "caught-errors",
    "error",
    "try/catch"
  ]
}
